keras/layers/reshaping/up_sampling2d.py

```diff
@@ -90,9 +91,20 @@ def __init__(self,
     super(UpSampling2D, self).__init__(**kwargs)
     self.data_format = conv_utils.normalize_data_format(data_format)
     self.size = conv_utils.normalize_tuple(size, 2, 'size')
-    if interpolation not in {'nearest', 'bilinear'}:
-      raise ValueError('`interpolation` argument should be one of `"nearest"` '
-                       f'or `"bilinear"`. Received: "{interpolation}".')
+    interpolations = {
+      'area': tf.image.ResizeMethod.AREA,
+      'bicubic': tf.image.ResizeMethod.BICUBIC,
+      'bilinear': tf.image.ResizeMethod.BILINEAR,
+      'gaussian': tf.image.ResizeMethod.GAUSSIAN,
+      'lanczos3': tf.image.ResizeMethod.LANCZOS3,
+      'lanczos5': tf.image.ResizeMethod.LANCZOS5,
+      'mitchellcubic': tf.image.ResizeMethod.MITCHELLCUBIC,
+      'nearest': tf.image.ResizeMethod.NEAREST_NEIGHBOR,
+    }
+    interploations_list = '`' + '`, `'.join(interpolations.keys()) + '`'
```

Comment:
Likewise

Reply:
Changed in [Fix string formatting](https://github.com/keras-team/keras/pull/16249/commits/aa8af10a160845df30a8b28470e866e02c68bdf5)